packages/client/tests/memory/_utils/generateMemoryTestClient.ts

```diff
@@ -35,6 +35,5 @@ export async function generateMemoryTestClient(testDir: MemoryTestDir) {
       edge: [__dirname.replace(/\\/g, '/'), '..', '..', '..', 'runtime', 'edge'].join('/'),
     },
     projectRoot: testDir.basePath,
-    dataProxy: !!process.env.TEST_DATA_PROXY,
```

Comment:
It looks like we are only testing the default (library) generated client in memory tests at the moment anyway.
It could be interesting to test more I think?